### PR TITLE
updating ember-cli and making it work again

### LINKED ISSUE #15: SASS Issues
I installed broccoli-sass as per the instructions but then I can't start the server because it complains about `tmp/app/styles/app.[scss|sass]` not existing in my project. I don't want to use sass in my own application.   Is there a workaround for this?

This seems like it might be a ember-cli issue as I can't see how an addon can dictate how to consuming application should format it's css code.

### PR DESCRIPTION
**Pull request by mansona**
Updating the ember-cli dep

fixing the broccoli-sass dependancy issues fixes https://github.com/Blooie/ember-cli-notifications/issues/15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,22 +21,21 @@
   "author": "Jonathan Steele",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.0.0",
-    "broccoli-sass": "^0.6.0",
-    "ember-cli": "0.2.0",
-    "ember-cli-app-version": "0.3.2",
-    "ember-cli-autoprefixer": "^0.3.0",
```
Comment (thedoktor) [inline package.json:28]: is this ok to be removed?

**Comment by thedoktor** [comment]
:+1: good to merge.

**Comment by mansona** [comment]
Merging!

**Comment by ynnoj** [comment]
🎉

**Pull request merged**